src/include/daos_srv/daos_engine.h

```diff
@@ -812,7 +812,7 @@ unsigned int dss_ctx_nr_get(void);
 
 /* Cache for container root */
 struct tree_cache_root {
-	struct btr_root	btr_root;
+	struct btr_root	*btr_root;
```

Comment:
This looks useless to me.

Reply:
hmm, this is used to free btr_root during tree destroy?

Reply:
dbtree_create_inplace() will use the root directly, instead of allocate a new one. so you have to allocate the root for dram tree.

Reply:
I mean the 'btr_root' in this data structure is useless.